nservicebus/dependency-injection/msdependencyinjection.md

```diff
@@ -21,10 +21,10 @@ snippet: msdependencyinjectionsharedcontainer
 
 ### DependencyLifecycle mapping
 
-[`DependencyLifecycle`](/nservicebus/dependency-injection/#dependency-lifecycle) maps to [`Microsoft.Extensions.DependencyInjection` service lifetimes](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.2) as follows:
+[`DependencyLifecycle`](/nservicebus/dependency-injection/#internally-managed-mode-built-in-default-container) maps to [`Microsoft.Extensions.DependencyInjection` service lifetimes](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-2.2) as follows:
```

Comment:
This comment is for this entire page, not this specific line. Since this page is about the community DI package which would no longer be needed for 7.2, shouldn't we be doing something more here? Either scoping this to before 7.2, or removing it entirely?

Reply:
I think we should remove it, @timbussmann already suggested this?

Reply:
Decide to remove and redirect to the main DI page

see f5b0cad0b4329db33d5115c745751d41a9c2317e